sourced/cool-oneliners/filesize.nu

```diff
@@ -0,0 +1,2 @@
+## show file sizes in current directory starting from the largest
+print (du --all | sort-by apparent | reverse | format filesize KB apparent)
```

Comment:
why the change from `ls -d` to `du --all`?

Reply:
Thanks for the catch. I got carried away a bit:) I have `nu v0.44.0` and `v0.32.0` and
1. There is no `apparent` column in `ls -d`
2. The command `help format filesize` has an example `du | format filesize apparent B`
3. This script is about sizes

However I agree I should just porting and not rewriting so maybe I should simply change `apparent` to `size` and keep `ls -d`